src/mrpro/data/_kdata/KData.py

```diff
@@ -250,3 +251,89 @@ def __repr__(self):
             f'{self.header}'
         )
         return out
+
+    def combine_coils(
```

Comment:
should this be compress_coil?
is coil combination not usually the application of the csms, and coilk compression what we do here?

(you are more versed in the terminology of MRI stuff, just asking)

Reply:
should we mention that this is "SCC" in the literature?

Reply:
scc == software channel compression? 

I added the corresponding paper although it is different as most of these approaches use image based ROIs to calculate the compression matrix...